### PR TITLE
Bug OCPBUGS-2832: mount cloud-event-proxy store to pvc for http

### PR DESCRIPTION
**Pull request by jzding**

### CODE REVIEW

File: bindata/linuxptp/ptp-daemon.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: linuxptp-daemon
-  namespace: openshift-ptp
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:6]: This should be always openshift-ptp

File: bindata/linuxptp/ptp-daemon.yaml [hunk 2]
```diff
@@ -127,7 +127,12 @@ spec:
             secretName: linuxptp-daemon-secret
         {{ if (eq .EnableEventPublisher true) }}
         - name: pubsubstore
+          {{ if (hasPrefix "http://" .EventTransportHost) }}
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:130]: has prefix http:// check might be an issue, also transport host can be given without http , And we need to support fo https .The best check will be to check for NOT amqp

File: bindata/linuxptp/ptp-daemon.yaml [hunk 3]
```diff
@@ -142,7 +147,7 @@ metadata:
   labels:
     name: monitor-ptp
   name: monitor-ptp
-  namespace: openshift-ptp
+  namespace: {{.Namespace}}
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:150]: do not change this. Its always openshift-ptp

File: bindata/linuxptp/ptp-daemon.yaml [hunk 4]
```diff
@@ -169,7 +174,7 @@ metadata:
   labels:
     name: ptp-monitor-service
   name:  ptp-monitor-service
-  namespace: openshift-ptp
+  namespace: {{.Namespace}}
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:177]: do not change this. It's always openshift-ptp

File: bindata/linuxptp/ptp-daemon.yaml [hunk 5]
```diff
@@ -185,7 +190,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: prometheus-k8s
-  namespace: openshift-ptp
+  namespace: {{.Namespace}}
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:193]: do not change this. It's always openshift-ptp

File: bindata/linuxptp/ptp-daemon.yaml [hunk 6]
```diff
@@ -202,7 +207,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: prometheus-k8s
-  namespace: openshift-ptp
+  namespace: {{.Namespace}}
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:210]: do not change this. It's always openshift-ptp

File: bindata/linuxptp/ptp-daemon.yaml [hunk 7]
```diff
@@ -219,7 +224,7 @@ metadata:
     prometheus: k8s
     role: alert-rules
   name: ptp-rules
-  namespace: openshift-ptp
+  namespace: {{.Namespace}}
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:227]: do not change this. It's always openshift-ptp

File: bindata/linuxptp/ptp-daemon.yaml [hunk 8]
```diff
@@ -246,3 +251,22 @@ spec:
           labels:
             severity: warning
         {{ end }}
+
+{{ if and (eq .EnableEventPublisher true) (hasPrefix "http://" .EventTransportHost) }}
```
Comment (aneeshkp) [inline bindata/linuxptp/ptp-daemon.yaml:255]: same here has HTTP is not bulletproof check

File: bindata/linuxptp/ptp-daemon.yaml (hunk 2 above)
Comment (jzding) [reply bindata/linuxptp/ptp-daemon.yaml]: updated

**Pull request merged**